templates/upload.tmpl

```diff
@@ -0,0 +1,106 @@
+            margin-bottom: 60px;
+            /* Margin bottom by footer height */
+        }
+
+        .footer {
+            position: absolute;
+            bottom: 0;
+            width: 100%;
+            height: 60px;
+            /* Set the fixed height of the footer here */
+            line-height: 60px;
+            /* Vertically center the text there */
+            background-color: #f5f5f5;
+        }
+    </style>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container">
+            <a class="navbar-brand" href="/upload">Plug</a>
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item active">
+                    <a class="nav-link" href="/upload">Upload</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/admin">Admin <span class="sr-only">(current)</span></a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <div class="jumbotron">
+        <div class="row justify-content-center">
+            <div class="col-lg-7">
+                <div class="jumbotron" style="max-width: 800px">
+                    <h1 class="display-3">Upload a Plug!</h1>
+                    <p class="lead">You will lose 1 drink credit in exchange for a 1000 view-limit of your plug.<br> Plugs must be 728x200 pixels and in PNG, or JPG format</p>
+                    <hr class="my-4">
+
+                    <div class="form-group">
+                        <input class="form-control-file" id="exampleInputFile" aria-describedby="fileHelp" type="file">
+                        <small id="fileHelp" class="form-text text-muted">Your Plug must be approved before it will appear for viewing. Any member of the following groups (drink, eboard, rtp) can do so via the admin page.</small>
+                    </div>
+                    <div class="float-right">
+                        <form action="/upload" method="post" enctype="multipart/form-data">
+                            <input class="btn btn-primary btn-lg" href="/upload" role="button" value="Upload" name="submit" type="submit"> </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="modal" id="agreementModal">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">User Agreement</h5>
+                </div>
+                <div class="modal-body">
+                    <p>The CSH Code Of Conduct Section 8 prohibits the sending of content that may harass others.</p>
+                    <p>Please review the <a href="http://latex.aslushnikov.com/compile?url=https%3A%2F%2Fraw.githubusercontent.com%2FComputerScienceHouse%2FCodeOfConduct%2Fmaster%2Fcsh-coc.tex" target="_blank">CSH Code Of Conduct</a> before uploading content to Plug.</p>
```

Comment:
I'd prefer if this was a link to the ComputerScienceHouse/CodeOfConduct repository rather than a link to a PDF on a third-party site.